packages/flutter/test/scheduler/scheduler_test.dart

```diff
@@ -245,6 +247,22 @@ void main() {
     warmUpDrawFrame();
     expect(scheduler.hasScheduledFrame, isTrue);
   });
+
+  test('The task was successfully executed after the animation', () async {
+    scheduler.schedulingStrategy = defaultSchedulingStrategy;
+    scheduler.debugResetEventLoop();
```

Comment:
I think we can remove this right? you get a new `scheduler` for each test.

It'd be best not to expose new API if we can avoid it.